src/openforms/formio/dynamic_config/__init__.py

```diff
@@ -44,7 +45,17 @@ def rewrite_formio_components_for_request(
     Loop over the formio configuration and inject request-specific configuration.
     """
     for component in configuration_wrapper:
-        register.update_config_for_request(component, request=request)
+        # prevent multiple overwrites of content components (bandaid for #2895)
+        if html_string := component.get("html", None):
+            soup = BeautifulSoup(html_string, "lxml")
+            tags = soup.find_all()
+            # find style attributes (not tags)
+            styles = [tag.get("style", None) for tag in tags]
+            nonce_missing = any(styles) and not all(
+                getattr(style, "nonce", None) for style in styles
+            )
+        if component["type"] != "content" or nonce_missing:
+            register.update_config_for_request(component, request=request)
```

Comment:
a component definition like `{"type": "content", "key": "content", "html": ""}` will cause this to crash because `nonce_missing` is an undefined variable